crates/fj-viewer/src/graphics/renderer.rs

```diff
@@ -34,6 +35,22 @@ pub struct Renderer {
 }
 
 impl Renderer {
+    /// Returns a new `Renderer`
+    ///
+    /// # Arguments
+    /// - `window` - a `winit::Window` with a surface to render onto.
```

Comment:
Almost true, but now quite. It's a `crate::window::Window`.

Reply:
Resolved by: https://github.com/hannobraun/Fornjot/pull/487/commits/3e86deb7e5482ed0dbf17a3c8d419fa77a429a5e.